services/security/security.go

```diff
@@ -268,8 +268,8 @@ type Identity struct {
 	Type *string `json:"type,omitempty"`
 	// Token
 	Token *string `json:"token,omitempty"`
-	// Certificate
-	Certificate *[]byte `json:"certificate,omitempty"`
+	// Certificates
+	Certificate *string `json:"certificates,omitempty"`
```

Comment:
is this the list of certificates that are held in map?

Reply:
This is the certificate passed in to the cloud agent, the map of client to cert is stored in the security.proto Identity struct.